File: workspace-servidor/flotix2021/src/main/java/com/flotix/response/bean/ServerResponseInit.java
package com.flotix.response.bean;

/**
 * Objeto de respuesta con el estado de la carga inicial
 * 
 * @author Flor
 *
 */
public class ServerResponseInit {

	private String msg = "";

	private ErrorBean error = new ErrorBean();

	public ServerResponseInit() {
		super();
	}

	public String getMsg() {
		return msg;
	}

	public void setMsg(String msg) {
		this.msg = msg;
	}

	public ErrorBean getError() {
		return error;
	}

	public void setError(final ErrorBean error) {
		this.error = error;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelResponse/ServerResponseInit.cs
namespace Flotix2021.ModelResponse
{
    class ServerResponseInit
    {
        private string _msg;

        public string msg
        {
            get { return _msg; }
            set { _msg = value; }
        }

        private ErrorBean _error;

        public ErrorBean error
        {
            get { return _error; }
            set { _error = value; }
        }
    }
}
